.github/workflows/tests-datapath-verifier.yaml

```diff
@@ -108,7 +105,7 @@ jobs:
           - kernel: '6.12-20241214.023514'
             ci-kernel: '61'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
-          - kernel: 'bpf-next-20241213.013358'
+          - kernel: '6.12-20241216.084703'
```

Comment:
Note that we already test with kernel 6.12 (one entry up). So I think this can just go away.